### PR TITLE
Added NRP verification proof

### PR DESCRIPTION
**Pull request by aritroCoder**

### CODE REVIEW

File: spec/data_flow_presentation_verify.md [hunk 1]
```diff
@@ -81,13 +81,21 @@ the presentation for the NRP being processed, plus the accumulator from
...
+$$\widehat{T_4} \leftarrow\left(\frac{e(\mathcal{G},\mathrm{acc})}{e(g,\mathcal{W})z}\right)^{c_H} \cdot e(\widetilde{h},\mathrm{acc})^{\widehat{r}}\cdot e(1/g,\widehat{h})^{\widehat{r'}}$$
+$$\widehat{T_5} \leftarrow D^{c_H}\cdot g^{\widehat{r}}\widetilde{h}^{\widehat{o'}}$$
+$$\widehat{T_6} \leftarrow  D^{\widehat{r''}}\cdot g^{-\widehat{m'}} \widetilde{h}^{-\widehat{t'}}$$
+$$\widehat{T_7} \leftarrow \left(\frac{e(pk\cdot\mathcal{G},\mathcal{S})}{e(g,g')}\right)^{c_H}\cdot e(pk\cdot \mathcal{G},\widehat{h})^{\widehat{r''}}\cdot e(\widetilde{h},\widehat{h})^{-\widehat{m'}}\cdot e(\widetilde{h},\mathcal{S})^{\widehat{r}}$$
+$$\widehat{T_8} \leftarrow \left(\frac{e(\mathcal{G},u)}{e(g,\mathcal{U})}\right)^{c_H}\cdot e(\widetilde{h},u)^{\widehat{r}}\cdot e(1/g,\widehat{h})^{\widehat{r'''}}$$
+
+Then all these values are added to $\hat{T}$. This is then added with the validity proof which when hashed with $\mathcal{C}$ and $n_1$(recieved from [[ref: holder]]) re constructs the challenge hash $\widehat{c_H}$
```
Comment (mikelodder7) [inline spec/data_flow_presentation_verify.md:95]:
```suggestion
Then all these values are added to $\widehat{T}$. This is then added with the validity proof which when hashed with $\mathcal{C}$ and $n_1$(recieved from [[ref: holder]]) re constructs the challenge hash $\widehat{c_H}$
```

File: spec/data_flow_presentation_verify.md [hunk 2]
```diff
@@ -81,13 +81,21 @@ the presentation for the NRP being processed, plus the accumulator from
...
+Now, if $\widehat{c_H} = c_H$ then the proof is valid.
```
Comment (mikelodder7) [inline spec/data_flow_presentation_verify.md:96]:
```suggestion
If $\widehat{c_H} = c_H$, then the proof is valid.
```

File: spec/data_flow_presentation_verify.md [hunk 3]
```diff
@@ -81,13 +81,21 @@ the presentation for the NRP being processed, plus the accumulator from
...
+
+The NRP is bound to the credential by the $\widehat{m_2}$ value that is present in the proof.
```
Comment (mikelodder7) [inline spec/data_flow_presentation_verify.md:98]:
```suggestion
The NRP is bound to the primary credential by the $\widehat{m_2}$ value that is presented in both proofs.
```

**Comment by aritroCoder** [comment]
Commited the changes

**Review by mikelodder7 — APPROVED** [review]

**Pull request merged**